spec

```diff
@@ -1053,10 +1053,9 @@ beginseg
 endseg
 
 beginseg
-    name "ovl_Bg_Jya_Lift"
-    include "build/src/overlays/actors/ovl_Bg_Jya_Lift/z_bg_jya_lift.o"
-    include "build/data/overlays/actors/z_bg_jya_lift.data.o"
-    include "build/data/overlays/actors/z_bg_jya_lift.reloc.o"
+	name "ovl_Bg_Jya_Lift"
```

Comment:
Not a huge deal, but for consistency I think we should keep this file using spaces instead of tabs.